### PR TITLE
Add/update executive director posting

### LINKED ISSUE #728: Add/update executive director posting in Careers section
We have the link for the combo Delphi+CMU executive director role!

https://cmu.wd5.myworkdayjobs.com/CMU/job/Pittsburgh-PA/Executive-Director-of-the-Delphi-Research-Group---Strategic-Coordinator-of-Public-Health-Research-Initiatives_2018952

We should:
- [ ] update the exec director entry on the careers page
- [ ] add a carousel entry for this position

### PR DESCRIPTION
**Pull request by christymelucci**
Add/update executive director posting in Careers section 

Closes #728 

- [ ]  update the exec director entry on the careers page
- [ ]  add a carousel entry for this position

### CODE REVIEW

**Comment by christymelucci** [comment]
@krivard 
Alt images to consider: 
https://drive.google.com/drive/folders/1ZTT5CG1zeKXUP089_60fLG_Oc6uWa7AK?usp=share_link

**Comment by christymelucci** [comment]
Closes #728

**Comment by christymelucci** [comment]
@krivard Open to comments. Happy to revise image.

**Comment by christymelucci** [comment]
@krivard  Can you indicate your preference for the blurb on the Delphi Careers page for the ED position?

- **[Text from CMU LinkiedIn:](https://www.linkedin.com/posts/kayla-graw-041055a3_research-leader-publichealth-activity-6998722549752700928-URfZ/?utm_source=share&utm_medium=member_desktop):** Carnegie Mellon Universityis looking for a strategic leader to join the Office of the Vice President of Research and the Delphi Research Group to lead all aspects of projects and initiatives that have the potential to transform public health research domestically and globally.
- **Text from CMU Careers:** Carnegie Mellon University is seeking a strong manager and strategist to lead all aspects of one of its major and ongoing public health research initiatives, the Delphi group, as well as coordinate other research activities that may craft the future of public health and have a real world impact in the next several years.

My preference is the text from CMU LinkedIn

**Comment by krivard** [comment]
@christymelucci I'm with you, the LinkedIn text is stronger

File: content/_index.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ description: Developing the Theory and Practice of Epidemiological Forecasting
 layout: landing
 
 carousel:
+  - pre: Delphi Careers
+    title: CMU Seeking an Executive Director for Delphi Research Group & Strategic Coordinator of Public Health Research Initiatives
+    ref: careers
+    alt: View Position
+    image: cmu-bldg-7-hero.jpg
   - pre: Delphi Careers
```
Comment (krivard) [inline content/_index.md:12]: We should nudge the general "We're Hiring!" carousel item into slot 3, otherwise we get two careers page items in a row

**Review by krivard — APPROVED** [review]
🚀 nice!

**Comment by christymelucci** [comment]
@krivard Quick check before I merge:
Retain or remove link to CMU in ED listing on Delphi Careers page?
https://deploy-preview-729--cmu-delphi-main.netlify.app/about/careers/

Does it complicate the listing or reinfoce that this is a CMU-wide position (as well as Delphi)?

**Comment by krivard** [comment]
Leave it in -- it's a joint position, and we don't have any others like it

**Pull request merged**